src/component/mxgraph/MxGraphConfigurator.ts

```diff
@@ -151,7 +182,7 @@ class BpmnMxGraph extends mxGraph {
   }
 
   private calculateFactorAndScale(up: boolean): [number, number] {
-    this.cumulativeZoomFactor *= up ? 1.2 : 0.8;
+    this.cumulativeZoomFactor *= up ? 1.25 : 0.8;
```

Comment:
:question: behaviour change, is this what we want? we see this change in test reference image that have been updated

Reply:
yes I have changed it, as it was behaving strangely with old values
imagine you start with scale 1 and you zoom in 2x you are in scale 1,44 now you zoom out 2x (thinking you will end with scale 1) and you land in 0.9216

with new value modifiers, you will always go back to 1

Reply:
Make sense, thanks for the explanations

Reply:
@aibcmars as this PR is about performance, I would extract this change in a dedicated PR.
We can add tests to ensure that performing zoom in then zoom out (or the opposite) restore the initial scale.
What do you think? I can handle it

Reply:
yes good idea.
done: #832